addons/packages/external-dns/0.8.0/test/Makefile

```diff
@@ -22,7 +25,7 @@ test: ## Run unit testing suite
 	@echo "TODO: implement unit tests"
 
 e2e-test: ## Run e2e testing suite
-	ginkgo -v ./addons/packages/external-dns/test/e2e
+	cd "${EXTERNAL_DNS_E2E_TEST_ROOT}" && ginkgo -v .
```

Comment:
This will probably be resolved when you fix conflicts, but we changed this to `CGO_ENABLED=0 go run github.com/onsi/ginkgo/ginkgo -v e2e`. This has a couple advantages, one you don't need ginkgo downloaded and two it will use the version of ginkgo that is in the go.mod (not that I've had any particular issues with using different versions of ginkgo).